payments/card/direct.md

```diff
@@ -0,0 +1,216 @@
+  you will have to redirect the payer.
+* When the payment is completed, the payer needs to be redirected back to your
+  merchant/webshop site.
+* Finally you make a `GET` request towards Swedbank Pay with the `paymentID`
+  received in the first step, which will return the purchase result.
+
+The sequence diagram below shows a high level description of a complete
+purchase, and the requests you have to send to Swedbank Pay. The links will take
+you directly to the corresponding API description.
+
+```mermaid
+sequenceDiagram
+    participant Payer
+    participant Merchant
+    participant SwedbankPay as Swedbank Pay
+
+    activate Payer
+    Payer->>+Merchant: start purchase
+    deactivate Payer
+    Merchant->>+SwedbankPay: POST /psp/creditcard/payments
+    deactivate Merchant
+    note left of Merchant: First API Request
+    SwedbankPay-->>+Merchant: rel: direct-authorization
+    deactivate SwedbankPay
+    Merchant->>+SwedbankPay: GET <payment.id>
+    deactivate Merchant
+    note left of Merchant: Second API request
+    SwedbankPay-->>+Merchant: rel: direct-authorization
+    deactivate SwedbankPay
+    Merchant-->>-Payer: display purchase result
+```
+
+## API Requests
+
+The API requests are displayed in the [purchase flow][purchase].
+You can [create a card `payment`][create-payment] with following `operation`
+options:
+
+* [`Purchase`][purchase]
+* [`Recur`][recur]
+* [`Payout`][payout]
+* [`Verify`][verify]
+
+Our `payment` example above uses the [`Purchase`][purchase] value which is
+explained below.
+
+### Purchase
+
+A `Purchase` payment is a straightforward way to charge the card of the payer.
+It is followed up by posting a capture, cancellation or reversal transaction.
+
+An example of an abbreviated `POST` request is provided below. Each individual
+Property of the JSON document is described in the following section.
+An example of an expanded `POST` request is available in the
+[other features section][purchase].
+
+{% include alert-risk-indicator.md %}
+
+{% include card-purchase.md %}
+
+## Type of authorization - Intent
+
+* **Authorization (two-phase):** If you want the credit card to reserve the
+  amount, you will have to specify that the `intent` of the `Purchase` is
+  `Authorization`. The amount will be reserved but not charged. You will later
+  (i.e. when you are ready to ship the purchased products) have to make a
+  [Capture][capture] or [Cancel][cancel] request.
+
+### General
+
+* *No 3-D Secure and card acceptance*: There are optional paramers that can be
+  used in relation to 3-D Secure and card acceptance. By default, most credit
+  card agreements with an acquirer will require that you use 3-D Secure for card
+  holder authentication. However, if your agreement allows you to make a card
+  payment without this authentication, or that specific cards can be declined,
+  you may adjust these optional parameters when posting in the payment.
```

Comment:
```suggestion
  you may adjust these optional parameters when creating the payment.
```
I found this sentence to be weirdly worded. `posting` is awkwardly technical term in a text that does not otherwise contain technical jargon.